### ISSUE TITLE
[5.0] Schemaorg plugins broken accessibility

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#41228]
### Steps to reproduce the issue
With the new plugins installed and enabled open an article, select a schema and check the html

### Expected result
Valid markup with no accessibility errors

#### Example from Bookposting
![image](https://github.com/joomla/joomla-cms/assets/1296369/b3ff2c74-3c54-4bf2-ae19-22dec60951f0)

Label for non existing input
![image](https://github.com/joomla/joomla-cms/assets/1296369/d9e68647-e98e-491a-8cd8-7e59d71cbfe6)

But it shouldnt be a label anyway
The fields in the subform-wrapper must be a fieldset with a legend as the first entry in the fieldset instead of the orphaned label above.

**Comment by bembelimen** [comment]
Is this schemaorg specific or a global bug in Joomla when using subforms?

**Comment by brianteeman** [comment]
It's just bad markup in the xml

**Comment by brianteeman** [comment]
basicaly these are fieldsets not subforms

**Comment by brianteeman** [comment]
Just rechecked and rechecked. 
The problem is 100% due to the schema plugin xml. There is no other xml form defintion in Joomla that has "nested" `<form>` elements.
In fact nested forms are very specifically mentioned 
> Warning: It's strictly forbidden to nest a form inside another form. Nesting can cause forms to behave unpredictably, so it is a bad idea
[source MDN](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_form_element)

(just because we call something a subform doesnt mean that it is)

**Comment by brianteeman** [comment]
I see that no one else really cares about valid html and/or accessibility

**Comment by brianteeman** [comment]
If they would just render as a fieldset then it would be fine. but no matter what I have tried it doesnt render as a fieldset. I must be missing somethign obvious

**Comment by Fedik** [comment]
This layout renders a "non multiple" subform: `layouts/joomla/form/field/subform/default.php`

**Comment by brianteeman** [comment]
yes i see that - which is wrong afaict and it should be a fieldset with a legend - see image above - but whenever I try to add it as a fieldest it doesnt render correctly. I must be missing something

**Comment by Fedik** [comment]
The legend will not show up as you expected. It can be only within that layout.

On your screenshot it is a field label, that rendered in `label` layout.

In result can be:
```
<label></label> => from field label

<fieldset>            |
   <legend></legend>  | => from field input
</fieldset>           |
```

**Comment by brianteeman** [comment]
that is what is wrong and breaks accessibility and html standards

**Comment by Fedik** [comment]
Following will not work?
```
<label for="blabla-input"></label> 

<fieldset id="blabla-input">            
   <legend aria-hidden></legend> 
</fieldset> 
```

If I right remamber we use the same for `checboxes`/`radio` field fieldset.

**Comment by brianteeman** [comment]
Yes thats one option. The "difference" is that for checkboxes/radio we are creating a fieldset out of a single field. In this case we should be rendering a fieldest containing several fields

**Comment by Fedik** [comment]
In theory it should be possible to do oposite: the label is hidden with `hiddenLabel="true"` in `xml`, and the legend visible.

**Comment by brianteeman** [comment]
it shouldnt be necessary. we already render multiple fieldsets elsewhere

**Comment by brianteeman** [comment]
https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/
